packages/wonder-blocks-color/docutils/swatch.js

```diff
@@ -0,0 +1,156 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+const {View} = require("wonder-blocks-core");
+const {HeadingLarge, LabelLarge, Body} = require("wonder-blocks-typography");
+const Color = require("../index").default;
+
+type Props = {
+    color: string,
+    name: string,
+    desc: string,
+    use: "text" | "icons" | null,
+    width: 256,
+    segments: 1 | 2 | 3,
+};
+
+const constants = {
+    segmentHeight: 64,
+};
+
+export default class Swatch extends React.Component<Props> {
```

Comment:
The implementation of this isn't the greatest but since this is only for the styleguide I'm hesitant to put too much time into it.